pkgs/desktops/deepin/dde-file-manager/default.nix

```diff
@@ -0,0 +1,235 @@
+{ stdenv, fetchFromGitHub, fetchpatch, pkgconfig, qmake, qttools,
+  qtmultimedia, qtx11extras, gnome3, gsettings-qt, dtkcore, dtkwidget,
+  deepin-movie-reborn, jemalloc, ffmpegthumbnailer, poppler, taglib,
+  libsecret, kcodecs, deepin-gettext-tools, dde-polkit-agent, deepin,
+  qt5integration, gtk2, qtsvg, file, avfs, polkit, polkit-qt,
+  deepin-shortcut-viewer, xdg-user-dirs, dde-qt-dbus-factory,
+  deepin-terminal, gst-plugins-good, mpv, zlib, libX11, gvfs, glib,
+  xorg, udisks2-qt5, dde-daemon, dde-dock, deepin-anything,
+  autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "dde-file-manager";
+  version = "4.8.4";
+
+  src = fetchFromGitHub {
+    owner = "linuxdeepin";
+    repo = pname;
+    rev = version;
+    sha256 = "0yhy36gkk5nyn56sn1v3hjri22nxs9k7raflbszahm574xpr1akb";
+  };
+
+  nativeBuildInputs = [
+    deepin.setupHook
+    qmake                   # build
+    qttools                 # build
+    pkgconfig               # build
+    deepin-gettext-tools    # build
+    autoPatchelfHook
```

Comment:
Having to resort to `autoPatchelfHook` is got to be the very last thing we'd want to do.

Why did we have to do this?

Reply:
It was needed when I started packaging this component some months ago. I do not remember exactly why. But I verified that it is not needed anymore.

Fixed.

Reply:
Thanks :+1: